tirtha_bk/tirtha/templates/tirtha/index.html

```diff
@@ -119,22 +119,18 @@ <h1>
             <button id="take-focus"></button>
             <section id="upload-area">
                 <h2>Contribute to Tirtha</h2>
-                <div id="g_id_onload"
-                    data-client_id="{{ GOOGLE_CLIENT_ID }}"
-                    data-context="use"
-                    data-ux_mode="popup"
-                    data-callback="onGoogleSignIn"
-                    data-auto_prompt="false">
-                </div>
-                <div class="upload-signin">
-                    <div class="g_id_signin"
-                        data-type="standard"
-                        data-shape="pill"
-                        data-theme="outline"
-                        data-text="signin_with"
-                        data-size="large"
-                        data-logo_alignment="left">
-                    </div>
+                <div class="button-container">
+                    {% if signin_msg == "Please sign in to upload images." %}
+                        <a id="signinButton" class="upload-signin" href="{% url 'signinGoogle' %}">
```

Comment:
You can move the if-else inside the `<a>` tag, as the props on `<a>` don't change. Only what goes inside of `<a>` changes.